dao/src/main/java/cz.muni.fi.pa165.soccermanager/entity/League.java

```diff
@@ -1,4 +1,147 @@
 package cz.muni.fi.pa165.soccermanager.entity;
 
+import javax.persistence.*;
+import java.util.Collections;
+import java.util.List;
+
+
+/**
+ * @author 476368 Iman Mehmandoust
+ * @version 10/27/2017.
+ * Class representing leauges.
+ * set of teams, country and name of the league
+ * You should use LeagueBuilder to create instances of League.
+ */
+
 public class League {
+
+    public static class LeagueBuilder {
+
+
+        private  String name;
+        private  String country;
+        private List<Match> matches;
+
+
```

Comment:
You should also add differenc constructor (for example public LeagueBuilder(String name, String country) makes sense)